### ISSUE TITLE
Unpublished Agenda

### ISSUE THREAD

**Issue by neilarellano** [datamade/la-metro-councilmatic#1107]
Greetings Team!

We had to unpublish the Agenda for the Ad Hoc 2028 Olympic & Paralympic Games Committee Meeting, We weren't supposed to publish this agenda since this meeting is cancelled.

Are we able to unpublish this from the site?

Also, do we have any logic set up to prevent blank agendas from being published? Or a way to unpublish an agenda on boardagendas site when it gets unpublished insite? Please advise.

**Comment by antidipyramid** [comment]
Hi @neilarellano, to confirm, are you referring to the cancelled 3/15 olympic committee meeting? [Currently, that meeting doesn't have an agenda on the site.](https://boardagendas.metro.net/event/ad-hoc-2028-olympic-d2c374a7de6f/)

**Comment by neilarellano** [comment]
> Hi @neilarellano, to confirm, are you referring to the cancelled 3/15 olympic committee meeting? [Currently, that meeting doesn't have an agenda on the site.](https://boardagendas.metro.net/event/ad-hoc-2028-olympic-d2c374a7de6f/)

Correct. I did notice that after posting this issue that the agenda seems to have been removed already from the site.

I guess this leaves us with just the idea of possibly setting up controls to prevent blank agendas from being published.

Thanks!

**Comment by antidipyramid** [comment]
@neilarellano We'll go through the scraper/site code to see what the current behavior is and if we need to make any changes.
